src/views/validated-designs/guide/index.tsx

```diff
@@ -4,56 +4,140 @@
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	headers: any // @TODO: fix type
+	markdown: {
+		mdxSource: MDXRemoteSerializeResult
+		title: string
+		description: string
+	}
+	headers: OutlineLinkItem[]
 	currentPageIndex: number
-	categorySlug: string
 	basePath: string
 	pages: HvdPage[]
 }
 
 export default function ValidatedDesignGuideView({
 	title,
-	mdxSource,
+	markdown,
 	headers,
 	currentPageIndex,
 	basePath,
 	pages,
 }: ValidatedDesignsGuideProps) {
+	const docsMdxComponents = getDocsMdxComponents('hcp')
+	const backBtnTitle = 'HVD Home'
+	const backBtnUrl = basePath
+
+	function renderPreviousLink() {
+		if (currentPageIndex !== 0 && pages[currentPageIndex - 1]) {
+			const page = pages[currentPageIndex - 1]
+
+			return (
+				<DirectionalLinkBox
+					label="Previous"
+					name={page.title}
+					href={page.href}
+					direction={'previous'}
+					ariaLabel={`Go to previous page: ${page.title}`}
+				/>
+			)
+		}
+	}
+
+	function renderNextLink() {
+		if (currentPageIndex !== pages.length - 1 && pages[currentPageIndex + 1]) {
+			const page = pages[currentPageIndex + 1]
+
+			return (
+				<DirectionalLinkBox
+					label="Next"
+					name={page.title}
+					href={page.href}
+					direction={'next'}
+					ariaLabel={`Go to next page: ${page.title}`}
+				/>
+			)
+		}
+	}
+
 	return (
 		<SidebarSidecarLayout
-			AlternateSidebar={() => (
-				<>
-					<InlineLink href={`${basePath}`}>
-						Back to Validated Designs landing
-					</InlineLink>
-					{pages.map((page: HvdPage, index: number) => (
-						<li key={page.slug}>
-							<InlineLink href={page.href}>{page.title}</InlineLink>
-							{index === currentPageIndex && <span> {'<- current'}</span>}
-						</li>
-					))}
-				</>
-			)}
-			sidebarNavDataLevels={[]}
-			// @TODO: move to using OutlineNavWithActiveLink
-			sidecarSlot={<OutlineNav items={headers} />}
+			sidebarNavDataLevels={[
+				{
+					backToLinkProps: {
+						text: backBtnTitle,
+						href: backBtnUrl,
+					},
+					title,
+					visuallyHideTitle: true,
+					showFilterInput: false,
+					showResourcesList: false,
+					children: (
+						<>
+							<SidebarSectionBrandedHeading text={title} theme={'hcp'} />
+							<SidebarHorizontalRule />
+							<SidebarNavList>
+								{pages.map((page: HvdPage, index: number) => {
+									return (
+										<SidebarNavHighlightItem
+											key={page.slug}
+											text={page.title}
+											theme={'generic'}
+											href={page.href}
+											isActive={index === currentPageIndex}
+										/>
+									)
+								})}
+							</SidebarNavList>
+						</>
+					),
+				},
+			]}
+			sidecarSlot={<OutlineNavWithActive items={headers} />}
 		>
 			<Head>
+				<title>{markdown.title}</title>
 				<meta name="robots" content="noindex, nofollow" />
+				<meta name="description" content={markdown.description} />
 			</Head>
-			<DevDotContent mdxRemoteProps={{ ...mdxSource }} />
+			<div className={s.mobileBackButton}>
```

Comment:
couldn't use `<MobileView>` here because the mobile breakpoint is slightly different then the `<SidebarSidecarLayout>`'s mobile breakpoint. 😢

Reply:
I don't see the back button in the [mobile mock](https://www.figma.com/file/s5mltxHQiTZ45PuqQ6b6JO/HVD-Phase-1?type=design&node-id=201-8272&mode=design&t=lEyXd5k6kahxwXJI-4). Was it decided on in conversation elsewhere?

Reply:
During a design session meeting with @kellyschummer we added it. @kellyschummer can we add that into the mock?